pkg/skaffold/build/custom/custom.go

```diff
@@ -68,9 +69,19 @@ func (b *ArtifactBuilder) Build(ctx context.Context, out io.Writer, a *latest.Ar
 
 func (b *ArtifactBuilder) retrieveCmd(out io.Writer, a *latest.Artifact, tag string) (*exec.Cmd, error) {
 	artifact := a.CustomArtifact
+
+	var cmd *exec.Cmd
 	split := strings.Split(artifact.BuildCommand, " ")
+	if len(split) > 1 {
```

Comment:
if we're going to check the platform here anyway, then is there any point in checking `if len(split) > 1`? functionally `exec.Command(artifact.BuildCommand)` and `exec.Command("sh", "-c", artifact.BuildCommand)` should be the same thing right?

Reply:
It's not the same. I'll had a comment